### PR TITLE
Update Linux MPI sample

### PR DESCRIPTION
**Pull request by xingwu1**

### CODE REVIEW

**Comment by xingwu1** [comment]
@matthchr @dlepow Could you review my PR?

**Comment by dlepow** [comment]
@xingwu1 - One suggestion would be to add a short README.md to this sample to give a brief explanation, or add a description to the higher level README.md at https://github.com/Azure-Samples/azure-batch-samples/blob/master/Python/Batch/README.md. I'm not an expert on the code but I appreciate the work on the simplified scenario. Thanks.

File: Python/Batch/article_samples/mpi/data/linux/openfoam/application-cmd [hunk 1]
```diff
@@ -1,43 +1,7 @@
-#!/usr/bin/env bash
-#Command script to run OpenFoam application on multiple machines as MPI task on Azure Batch.
-#Usage: application [nodes] [cores_per_node] [CFD iterations]
-
-#prepare environment variables for intel mpi and openfoam
```
[the remark this answers is not in the document]
Comment (xingwu1) [reply Python/Batch/article_samples/mpi/data/linux/openfoam/application-cmd]: we are removing everything releated to openform. I don't think we should keep the comment.

File: Python/Batch/article_samples/mpi/data/linux/openfoam/application-cmd [hunk 2]
```diff
@@ -1,43 +1,7 @@
-#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (xingwu1) [reply Python/Batch/article_samples/mpi/data/linux/openfoam/application-cmd]: same reason of above.

**Comment by xingwu1** [comment]
@dlepow I added a README for the article. 
@matthchr I update the sample.

File: Python/Batch/article_samples/mpi/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+### Setting up the Python environment
+In order to run the samples, you will need a Python interpreter compatible
+with version 2.7 or 3.3+. You will also need to install the
+[Azure Batch](https://pypi.python.org/pypi/azure-batch) and
+[Azure Storage](https://pypi.python.org/pypi/azure-storage) python packages.
+Installation can be performed using the [requirements.txt](./requirements.txt)
+file via the command `pip install -r requirements.txt`
+
+### MPI sample
+The MPI sample is an introduction to use Batch service to run MPI command under
```
Comment (matthchr) [inline Python/Batch/article_samples/mpi/README.md]: The MPI sample is an introduction to running an MPI command in Azure Batch in a Linux environment.  It creates a pool with RDMA enabled virtual machines (You can use STANDARD_A8 to replace the virtual machine size if you don't have enough core quota). It submits a job to the pool, and then submits a task which performs a simple MPI pingpong command. The files required by the task are automatically uploaded to Azure Storage and then downloaded onto the nodes via Batch resource files. When the task is done, the console output of the MPI pingpong command is uploaded back to Azure Storage. 

For more details on MPI/RDMA, visit [here](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/classic/rdma-cluster).

**Pull request merged**